config/passport.js

```diff
@@ -0,0 +1,79 @@
+const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
+const LocalStrategy = require('passport-local').Strategy;
+const BearerStrategy = require('passport-http-bearer').Strategy;
+const configAuth = require('./settings');
+
+module.exports = function(passport, models) {
+
+	const localUser = models.user_account;
+
+	passport.serializeUser(function(user, done){
+		done(null, user);
+	});
+
+	passport.deserializeUser(function(user, done){
+		done(null, user);
+	});
+
+	passport.use(new GoogleStrategy({
+			clientID: configAuth.googleAuth.clientID,
```

Comment:
like @visitha & I said in the old PR, please make sure **ALL** of your files are **properly** and **evenly** spaced.
it is to your benefit to introduce some linting tools to auto check all the basics for you - saves us time, and forces you to code properly.
google has a detailed [JS style guide](https://google.github.io/styleguide/jsguide.html) that is yet to be updated with ES6 standards, and airbnb has an awesome [ES6 style guide](https://github.com/airbnb/javascript) as well. I'm pretty sure there are other ones available, pick one that best suits your coding style